app/controllers/katello/remote_execution_controller.rb

```diff
@@ -43,7 +43,7 @@ def hosts
       end
 
       def errata_inputs
-        if params[:install_all]
+        if params[:install_all] == 'true'
```

Comment:
Using  ::Foreman::Cast.to_bool(params[:install_all])  would be preferred here

Reply:
I know such a method existed :smile: 
Thanks for the hint.